development/kops/gather_logs.sh

```diff
@@ -42,6 +42,7 @@ function log_dump_custom_get_instances() {
     # Container logs are symlinks and need a slightly different chmod
     # log-dump-ssh comes the upstream log-dump.sh script
     for node in $nodes ; do log-dump-ssh $node "sudo chmod a+r /var/log/containers/*" || true ; done
+    for node in $nodes ; do log-dump-ssh $node "sudo chmod +x /var/log/pods" || true ; done
```

Comment:
does this end up running the log dump twice? its probably fine if it does and maybe im misremembering how this script works.

Reply:
no, this runs the chmod and nothing else, the function name i think is confusing because they tacked on a log-dump- prefix